src/overlays/actors/ovl_En_Shn/z_en_shn.h

```diff
@@ -7,10 +7,58 @@ struct EnShn;
 
 typedef void (*EnShnActionFunc)(struct EnShn*, GlobalContext*);
 
+typedef enum {
+    /* 00 */ SHN_LIMB_NONE,
+    /* 01 */ SHN_LIMB_WAIST,
+    /* 02 */ SHN_LIMB_LEG_LEFT,
+    /* 03 */ SHN_LIMB_SHIN_LEFT,
+    /* 04 */ SHN_LIMB_FOOT_LEFT,
+    /* 05 */ SHN_LIMB_LEG_RIGHT,
+    /* 06 */ SHN_LIMB_SHIN_RIGHT,
+    /* 07 */ SHN_LIMB_FOOT_RIGHT,
+    /* 08 */ SHN_LIMB_TORSO,
+    /* 09 */ SHN_LIMB_SHOULDER_LEFT,
+    /* 10 */ SHN_LIMB_FOREARM_LEFT,
+    /* 11 */ SHN_LIMB_HAND_LEFT,
+    /* 12 */ SHN_LIMB_SHOULDER_RIGHT,
+    /* 13 */ SHN_LIMB_FOREARM_RIGHT,
+    /* 14 */ SHN_LIMB_HAND_RIGHT,
+    /* 15 */ SHN_LIMB_HEAD,
+    /* 16 */ SHN_LIMB_MAX,
+} ObjectShnLimbs;
+
 typedef struct EnShn {
-    /* 0x0000 */ Actor actor;
-    /* 0x0144 */ EnShnActionFunc actionFunc;
-    /* 0x0148 */ char unk_148[0x1A8];
+    /* 0x000 */ Actor actor;
+    /* 0x144 */ EnShnActionFunc actionFunc;
+    /* 0x148 */ SkelAnime skelAnime;
+    /* 0x18C */ UNK_TYPE1 pad18C[0x4C];
+    /* 0x1D8 */ u16 unk1D8;
+    /* 0x1DA */ u16 unk1DA;
+    /* 0x1DC */ UNK_TYPE4* unk1DC;
+    /* 0x1E0 */ s32 unk1E0;
+    /* 0x1E4 */ Player* unk1E4;
+    /* 0x1E8 */ Vec3f unk1E8;
+    /* 0x1F4 */ Vec3s unk1F4;
+    /* 0x1FA */ Vec3s unk1FA;
+    /* 0x200 */ UNK_TYPE1 pad200[0x5A];
+    /* 0x25A */ Vec3s unk25A;
+    /* 0x260 */ UNK_TYPE1 pad260[0x5A];
```

Comment:
`unk1FA` and `unk25A` are `jointTable` and `morphTable` respectively. You can know that because those two are passed to `SkelAnime_InitFlex`.
Also, those are always arrays of `Vec3s`, not single elements. The size usually is the `LIMB_MAX` from the object's limb enum.
```suggestion
    /* 0x1FA */ Vec3s jointTable[SHN_LIMB_MAX];
    /* 0x25A */ Vec3s morphTable[SHN_LIMB_MAX];
```